src/ln/functional_tests.rs

```diff
@@ -5353,17 +5353,39 @@ fn test_update_fulfill_htlc_bolt2_after_malformed_htlc_message_must_forward_upda
 	check_added_monitors!(nodes[1], 1);
 }
 
+#[test]
+#[should_panic]
```

Comment:
Hmm we currently panic in send_commitment, but imo it has to be modified to return ChannelError::Ignore like in send_htlc.

Reply:
Agreed - will work on the change - thanks